recipes/geometric/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "geomeTRIC" %}
+{% set version = "0.8.8" %}
+
+# openssl sha256 filename
+{% set sha256 = "45ebea62a43045acd71d41516d732cb76bf65d760571a30454d4ff252dc1e10a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/leeping/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - numpy>=1.11
```

Comment:
There should be a space, `numpy >=1.11`, but you could also use `pin_compatible()` https://conda.io/docs/user-guide/tasks/build-packages/variants.html?highlight=pin_compatible#pinning-at-the-recipe-level